cpp/data/sql_updates/ub_tools.24

```diff
@@ -0,0 +1,10 @@
+-- We rename metadata_presence_tracer.zeder_journal_id to journal_id
+-- and add the new column metadata_presence_tracer.marc_subfield_code.
+-- Constraints affected by the rename are also being updated.
+TRUNCATE TABLE metadata_presence_tracer;
+ALTER TABLE metadata_presence_tracer DROP CONSTRAINT metadata_presence_tracer_zeder_journal_id;
+ALTER TABLE metadata_presence_tracer DROP CONSTRAINT metadata_presence_tracer_journal_id_and_marc_field_tag;
+ALTER TABLE metadata_presence_tracer ADD COLUMN marc_subfield_code CHAR(1) NOT NULL AFTER marc_field_tag;
+ALTER TABLE metadata_presence_tracer CHANGE COLUMN zeder_journal_id journal_id INT(11) UNSIGNED;
+ALTER TABLE metadata_presence_tracer ADD CONSTRAINT metadata_presence_tracer_journal_id FOREIGN KEY (journal_id) REFERENCES zeder_journals (id) ON DELETE CASCADE ON UPDATE CASCADE;
+ALTER TABLE metadata_presence_tracer ADD CONSTRAINT metadata_presence_tracer_journal_id_and_marc_field_tag UNIQUE (journal_id, marc_field_tag);
```

Comment:
Also add `marc_subfield_code` to the UNIQUE constraint here & change the name

Reply:
Done.  Please note that constraint names can be at most 64 characters long.